packages/p2p/src/components/my-ads/create-ad-add-payment-method-modal.jsx

```diff
@@ -5,7 +5,7 @@ import { Button, Icon, MobileFullPageModal, Modal, Text } from '@deriv/component
 import { isMobile } from '@deriv/shared';
 import { useStores } from 'Stores';
 import { localize, Localize } from 'Components/i18next';
-import AddPaymentMethod from '../my-profile/payment-methods/add-payment-method/add-payment-method.jsx';
+import AddPaymentMethod from 'Pages/my-profile/payment-methods/add-payment-method/add-payment-method.jsx';
```

Comment:
```suggestion
import AddPaymentMethod from 'Pages/my-profile/payment-methods/add-payment-method';
```

Could you remove the file extensions where theres already index for them like for add-payment-method